eth/downloader/peer.go

```diff
@@ -31,6 +31,9 @@ import (
 	"gopkg.in/fatih/set.v0"
 )
 
+// The impact a single measurement has on a peer's final throughput value.
+const throughputImpact = 0.1
```

Comment:
Hmmm, although I generally agree, perhaps this is better here. The reason is that it's only used in this single file. Moving it to somewhere else would move it seriously out of scope.